frontend/src/components/navbar/TopNavbar.vue

```diff
@@ -130,16 +190,93 @@ export default {
         },
         toggleMenu () {
             this.navIsActive = !this.navIsActive;
+        },
+        toggleNotifications() {
+            this.isNewNotifications = false;
+            this.notificationsDisplay = !this.notificationsDisplay;
+        },
+        hideNotifications() {
+            this.notificationsDisplay = false;
+        },
+        notificationComponent: function(notification) {
+            switch (notification.type) {
+                case 'Hedonist\\Notifications\\LikeReviewNotification':
```

Comment:
`Hedonist\\Notifications\\LikeReviewNotification` must be const values in notificationService